honeypot/Datadog.Dependabot.Honeypot.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp3.1</TargetFrameworks>
+    <RootNamespace>Datadog.Dependabot.Honeypot</RootNamespace>
```

Comment:
Why `Honeypot`?

Reply:
It seemed like an apt name.
https://en.wikipedia.org/wiki/Honeypot_(computing)
It's not an attacker, but it isn't a REAL project: It's a fake project designed to attract the dependabot.

Reply:
>designed to attract the dependabot.

Ah, I didn't think of it that way. Thanks.